test/app.createRouter.js

```diff
@@ -0,0 +1,37 @@
+var express = require('../');
+
+describe('app.createRouter()', function(){
+
+  it('should create a router that inherits settings from app.settings', function(done){
```

Comment:
Since this is not an async test, you don't need to use `done` here and just throw the errors